src/canvas.tsx

```diff
@@ -352,13 +351,33 @@ export const useCanvas = (props: UseCanvasProps): DomEventHandlers => {
 
   /**  Calculates click deltas */
   const calculateDistance = useCallback((event: DomEvent) => {
-    let dx = event.clientX - state.current.initialClick[0]
-    let dy = event.clientY - state.current.initialClick[1]
+    const dx = event.clientX - state.current.initialClick[0]
+    const dy = event.clientY - state.current.initialClick[1]
     return Math.round(Math.sqrt(dx * dx + dy * dy))
   }, [])
 
   const hovered = useMemo(() => new Map<string, DomEvent>(), [])
 
+  const handlePointerCancel: any = useCallback((event: DomEvent, hits?: Intersection[], prepare = true) => {
+    state.current.pointer.emit('pointerCancel', event)
+    if (prepare) prepareRay(event)
+    // commenting this out as I believe it is unneccessary and causes a recursive dependency
+    // if (!hits) hits = handleIntersects(event, () => null)
```

Comment:
I'm going to look further into this before merging, but I don't believe this invocation is actually necessary.